std/uni.d

```diff
@@ -7318,7 +7332,7 @@ if (isInputRange!Range && is(Unqual!(ElementType!Range) == dchar))
     assert("".byGrapheme.byCodePoint.equal(""));
 
     string text = "noe\u0308l";
-    static assert(is(typeof(text.byCodePoint) == string));
+    static assert(!__traits(compiles, (){ "noe\u0308l".byCodePoint.length; }));
```

Comment:
This doesn't compile for another reason (which is that the expression inside the delegate literal is invalid). 
You probably meant 
```d
static assert(!__traits(compiles, "noe\u0308l".byCodePoint.length));
```
or
```d
static assert (!is(typeof((){return "noe\u0308l".byCodePoint.length;})));
```

Reply:
Yeah, my bad